src/figuresClasses.ts

```diff
@@ -1,19 +1,101 @@
-export interface Figure {
+const ERROR_MESSAGE: string = 'Value cannot be <= 0!';
```

Comment:
```suggestion
const ERROR_MESSAGE = 'Value cannot be <= 0!';
```

Reply:
You don't need to set type manually. Also as previous mentor mention, you need to give more descriptive  name to this variable